tutorials/FITS-images/FITS-images.ipynb

```diff
@@ -392,43 +699,55 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "Diir_0hlgEIy"
+   },
    "source": [
     "## Exercises"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "wpPk3Zv6gEIy"
+   },
    "source": [
-    "Determine the mean, median, and standard deviation of a part of the stacked M13 image where there is *not* light from M13.  Use those statistics with a sum over the part of the image that includes M13 to estimate the total light in this image from M13."
+    "Determine the mean, median, and standard deviation of the stacked M13 image data.  Add corresponding keywords 'MEAN', 'MEDIAN', and 'STANDARD DEVIATION' to the primary HDU in the outfile you just writed to, and store the statistics you calculated in the keywords created in the header. See the [Editing a FITS Header tutorial](https://learn.astropy.org/tutorials/FITS-header.html) to learn how to add the keywords. "
```

Comment:
This is a substantial change to the exercise but I think the new one is better 😀 . The old one was more an exercise is using numpy than in using FITS.